libs/community/tests/unit_tests/callbacks/test_imports.py

```diff
@@ -29,4 +29,6 @@
 
 
 def test_all_imports() -> None:
+    """Test that __all__ is correctly set."""
     assert set(__all__) == set(EXPECTED_ALL)
```

Comment:
I guess, we don't need the EXPECTED_ALL anymore, no?

Reply:
We usually maintain a second list of hard-coded imports in unit tests to catch accidental breaking changes (e.g., removal of an import)

Reply:
We could remove in this case and just make sure that set(__all__) == the keys from the module. I don't have a strong preference